packages/frontend/src/hooks/useCookies/canStoreCookies.ts

```diff
@@ -0,0 +1,8 @@
+import { CookieNames } from "."
```

Comment:
These are not hooks. Can you move all the functions to utils folder in `cookies.ts`?